src/diffusers/loaders.py

```diff
@@ -1018,24 +1040,94 @@ def lora_state_dict(
         else:
             state_dict = pretrained_model_name_or_path_or_dict
 
+        # TODO(SAYAK) - the following mapping is needed whenever LoRAs are published
+        # in original SGM format (this was also meant here: https://github.com/huggingface/diffusers/pull/4287#discussion_r1275044911)
+        if unet_config is not None:  # need better chec
+            # use unet config to remap block numbers
+            state_dict = cls._map_sgm_blocks_to_diffusers(state_dict, unet_config)
+
+        # Renaming should be correct now!
+
         # Convert kohya-ss Style LoRA attn procs to diffusers attn procs
-        network_alpha = None
+        network_alphas = None
         if all((k.startswith("lora_te_") or k.startswith("lora_unet_")) for k in state_dict.keys()):
-            state_dict, network_alpha = cls._convert_kohya_lora_to_diffusers(state_dict)
+            # TODO(SAYAK) - here we're now missing a couple of keys notably:
```

Comment:
@patrickvonplaten did you mean to complete this comment?

Reply:
Think that's the next step to get SDXL LoRA working